src/pudl/transform/ferc1.py

```diff
@@ -265,7 +277,9 @@ class Config:
     )
     rename_columns_instant_xbrl: RenameColumns = RenameColumns()
     rename_columns_duration_xbrl: RenameColumns = RenameColumns()
-    wide_to_tidy_xbrl: WideToTidyXbrl = WideToTidyXbrl()
+    wide_to_tidy: WideToTidy = WideToTidy(
+        dbf=WideToTidyDataSource(), xbrl=WideToTidyDataSource()
+    )
```

Comment:
Is there a reason that this combo xbrl + dbf transform is following a different pattern than the DBF/XBRL or instant/duration `RenameColumns` functions / params? Why is it necessary to create a composite parameter type that includes two instances of the `WideToTidyDataSource` parameter type? IIRC in the other transforms that get reused for multiple subsets of the data, we've looked up the appropriate parameter using the data source ID, rather than creating additional layers of composite parameters.

Reply:
as i said in my PR comment, I mirrored this transformer off of `Ferc1RenameColumns`. I did this because we are applying the same transform with different steps of parameters depending on the different source of the ferc1 data